Chapter11/das_node.erl

```diff
@@ -0,0 +1,39 @@
+-module(das_node).
+-export([create/1, address_loop/1]).
```

Comment:
now I'm confused why das node is needed? it seems like you can just replace its functionality by adding some functionality to balancer node, so that write/read/delete messages do not go through several processes, but rather directly from balancer to node.

Reply:
Ahh yeah I see what you mean, I've deleted the das_node nodes and shifted there functionality to balancer. I also changed how the lookup works, so instead of the balance node waiting for a response from the a db_node, the db_node sends the lookup response straight to the das:lookup function :)

Reply:
The lookup change should reduce bottlenecks in balancer